### PR TITLE
Tlportal 175 add check url for monitoring

### PR DESCRIPTION
**Pull request by dlhaines**

### CODE REVIEW

File: server/courselist.rb [hunk 1]
```diff
@@ -525,7 +533,7 @@ def self.getURLUniqname(url)
       return "format missing or not supported: [#{format}]"
     end
 
-     #logger.debug "#{__LINE__}: course_data.value_as_json: "+course_data.value_as_json.inspect
+    #logger.debug "#{__LINE__}: course_data.value_as_json: "+course_data.value_as_json.inspect
```
Comment (zqian) [inline server/courselist.rb]: Remove this line?

File: server/local/studentdashboard-LOCALHOST.yml [hunk 2]
```diff
@@ -55,7 +59,7 @@ latte_admin_group: TL-Latte-admin-test
 ## from local files.  Configuration is documented below.
 
 ### file based data provider value:
-data_provider_file_directory: server/test-files/courses
+data_provider_file_directory: server/test-files
```
Comment (zqian) [inline server/local/studentdashboard-LOCALHOST.yml]: The  server/test-files/courses/ststvii.json is still inside courses folder. Is it OK to omit "courses" here in the data_provider_file_directory setting?

File: server/courselist.rb [hunk 3]
```diff
@@ -77,6 +77,9 @@ class CourseList < Sinatra::Base
 
   @@default_term = 2010
 
+  @@check_uniqname = "ststvii"
```
Comment (bkirschn) [inline server/courselist.rb]: Shouldn't this be pulled from the yml file (defined below)?

File: server/courselist.rb (hunk 1 above)
Comment (dlhaines) [inline server/courselist.rb]: Sure.

File: server/local/studentdashboard-LOCALHOST.yml (hunk 2 above)
Comment (dlhaines) [inline server/local/studentdashboard-LOCALHOST.yml]: No.  The test files now come from two different sub-directories so this is specifying the parent directory.

File: server/courselist.rb (hunk 3 above)
Comment (dlhaines) [inline server/courselist.rb]: These are defaults.  In practice the value is pulled from the yml file.

**Pull request merged**